packages/private-build-infra/src/stripped-build-plugins.js

```diff
@@ -22,7 +21,7 @@ module.exports = function (environment, app, compatVersion) {
     plugins.push([FilterImports, { imports: filteredImports }]);
   }
 
-  plugins.push([TransformBlockScoping, { throwIfClosureRequired: true }], ...DebugMacros);
+  plugins.push(...DebugMacros);
```

Comment:
@runspired This is the right removal, right?

Reply:
No, though it's likely also safe to remove this now. There's several GitHub actions/test run scenario infra bits that expect an IE11 compat build.